MonoGame.Framework/Input/GamePadButtons.cs

```diff
@@ -2,11 +2,14 @@
 // This file is subject to the terms and conditions defined in
 // file 'LICENSE.txt', which is part of this source code package.
 
+using System;
+
 namespace Microsoft.Xna.Framework.Input
 {
     /// <summary>
     /// A struct that represents the current button states for the controller.
     /// </summary>
+    [Serializable]
```

Comment:
Ideally yes.  We should cross reference it and see what it did in these cases.

If the type is an XNA type and XNA didn't support serialization on we should decide on a case by case basis if we should add it.  I figure it may be ok to add in most cases.

My concern is trying to serialize something that is considered internal implementation details that we might want to change in the future.